solidity/.gitignore

```diff
@@ -1,3 +1,5 @@
 node_modules/
 build/
 .python-version
+.zos.session
+package-lock.json
```

Comment:
Why are we ignoring the `package-lock.json` file? I believe we want to lock the versions from a consistency and security perspective.

Reply:
We're never actually installing anything in this directory.